tinyagentos/agent_git.py

```
@@ -0,0 +1,164 @@
+        raise RuntimeError(f"git rev-parse HEAD failed: {head_out}")
+    head_sha = head_out.strip()
+    if sha == head_sha:
+        return "noop"
+    await git_rev_parse(container, sha)
+    if not await git_merge_base_is_ancestor(container, sha):
+        raise NotAncestorError(f"{sha} is not an ancestor of HEAD")
+    script = (
+        "dirty=$(git -C /root status --porcelain); "
+        'test -z "$dirty" && git -C /root reset --hard ' + sha
+    )
+    rc, out = await exec_in_container(
```

Ruff (0.16.3)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)